src/vs/workbench/contrib/chat/common/chatServiceImpl.ts

```diff
@@ -180,7 +180,7 @@ export class ChatService extends Disposable implements IChatService {
 	private saveState(): void {
 		const liveChats = Array.from(this._sessionModels.values())
 			.filter(session =>
-				this.shouldSaveToHistory(session.sessionId) && (session.initialLocation === ChatAgentLocation.Panel || session.initialLocation === ChatAgentLocation.Editor));
+				!session.inputType && (session.initialLocation === ChatAgentLocation.Panel || session.initialLocation === ChatAgentLocation.Editor));
```

Comment:
Is it right that `this.shouldSaveToHistory` isn't called here anymore? Is that function still needed?